### ISSUE TITLE
Backend doesn't allow to read Config out of the box

### ISSUE THREAD

**Issue by mgttlinger** [obsidiansystems/obelisk#865]
I might be doing things the wrong way but it seems that the Frontend handles a `ConfigsT` in its transformer stack while the Backend doesn't. I can of course handle it myself but it feels like a bug or at least kind of an oddity that the Frontend allows reading from the config but the backend does not out of the box.

**Comment by alexfmpe** [comment]
FWIW there's `Obelisk.ExecutableConfig.Lookup.getConfigs :: IO (Map Text ByteString)`. 
Not sure how that relates to `Obelisk.Configs.getConfigs :: HasConfigs m => m (Map Text ByteString))`, but both show up on `ob hoogle` when searching for `getConfigs`.

**Comment by mgttlinger** [comment]
Yes, that is what you call the `runConfigsT` with, but it feels strange that I have to do this in the Backend while the Frontend does it for me.

**Comment by mgttlinger** [comment]
If this is not the intended behaviour I can gladly send a PR changing that.
